api/src/api/teams.ts

```diff
@@ -57,6 +58,76 @@ router.post(
   }),
 );
 
+// Create many new teams
+router.post(
+  '/many',
+  requireAdmin,
+  errorWrap(async (req: Request, res: Response) => {
+    const newTeams: Partial<ITeam>[] = req.body.teams;
+
+    const createdTeams = await Promise.all(
+      newTeams.map(async (team) => Team.create(team)),
```

Comment:
You can probably remove `async` from this inner function since you're not using `await` in it. I may be wrong though.